### PR TITLE
Handle arenas with numbers of corners other than 4

### PR DESCRIPTION
**Pull request by PeterJCLaw**
The approach for the arena displays is a bit blunt as ideally we'd automatically adapt to the right number of corners.

The other displays are somewhat easier to cope with as they're just tables, so they do now adapt to the given number of corners.

### CODE REVIEW

File: components/sr-arena/component.html [hunk 1]
```diff
@@ -34,42 +34,47 @@
...
                     corners[i].arena = this.arena;
                 }
             },
             maincornerChanged: function() {
-                var otherCorners = [0, 1, 2, 3];
-                otherCorners.splice(this.maincorner, 1);
-
                 var corners = this.shadowRoot.querySelectorAll('sr-corner');
-                for (var i = 0; i < corners.length; i++) {
-                    if (i == 0) {
```
Comment (thomasleese) [inline components/sr-arena/component.html:61]: Should probably use `===` here.

**Review by thomasleese — APPROVED** [review]
Looks good to me!

File: components/sr-arena/component.html (hunk 1 above)
Comment (PeterJCLaw) [reply components/sr-arena/component.html:61]: Did you mean to comment on the code being removed?

Comment (thomasleese) [reply components/sr-arena/component.html:61]: Oh, I actually thought I had removed this before submitting the review when I realised...

**Pull request merged**